introduction/InstruDictator.html

```diff
@@ -1,51 +1,95 @@
 {{ block title }}Instructions{{ endblock }}
 
+{{ block style }}
+
+<style>
+    /* (A) SHARED CLASS */
+    .blue_box {
+    padding: 10px;
+    border: 2px solid dodgerblue;
+    height: 40px;
+    width: 70px;
+    border-radius: 5px;
+    background: dodgerblue;
+    color: white;
+    }
+
+    div.centered {
+    width: 500px;
+    height: 400px;
+    top: 50%;
+    left: 50%;
+    position: absolute;
+    margin-top: -200px;
+    margin-left: -250px;
+}
+</style>
+
+{{ endblock }}
+
 {{ block content }}
 
 <div>
     <p>
         You are the <b>Decider</b>. This means that in every interaction you have a decision to make.
```

Comment:
From reading this, it was not entirely clear to me whether this means I will really be the decider in the experiment, or whether this is just a hypothetical statement for the sake of the instruction. 
Perhaps it would be more clear if you said "For this experiment, you have been randomly assigned the role of the <b>Decider</b>.